resources/js/visual-editor/site-editor/entity-browser.css

```
@@ -0,0 +1,174 @@
+.ap-site-editor__entity-browser-error {
+    background: var(--ap-site-editor-error-bg, #fef2f2);
+    border-top: 1px solid var(--ap-site-editor-error-border, #fecaca);
+    border-bottom: 1px solid var(--ap-site-editor-error-border, #fecaca);
+    color: var(--ap-site-editor-error-fg, #991b1b);
+}
+
+.ap-site-editor__entity-browser-retry {
+    margin-top: 4px;
+    appearance: none;
+    background: transparent;
+    border: 1px solid currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)